src/routes/(site)/shows/+page.server.ts

```diff
@@ -43,9 +46,12 @@ export const load: PageServerLoad = async function ({ locals, url, setHeaders })
 		throw redirect(302, `/shows?${params_string}`);
 	}
 
+	const { value: shows } = shows_result;
+	const total_show_count = count_result.status === 'fulfilled' ? count_result.value : 42;
```

Comment:
What is the 42?

Reply:
It is only a default value if the count promise is rejected. Not graceful, but I didn't want to make significant changes.

Reply:
Gotcha, could we just change this all to promise.all? If the DB query fails, we are in bigger trouble and should just let the error page show.

Reply:
Of course! Also done 👍